### ISSUE TITLE
Minor issue building with wxWidgets 3.1.1 on Windows 10

### ISSUE THREAD

**Issue by grumpy235** [moneymanagerex/moneymanagerex#1819]
**Your OS:**
- [X] Windows
- [ ] Mac OSX
- [ ] Linux 
- [ ] Android -> please report bugs to moneymanagerex/android-money-manager-ex repo!

**MMEX version:**
- [ ] 1.4.x
- [ ] 1.3.x
- [ ] 1.2.x
- [ ] 1.1 or older
- [X] development version (git master branch)

I compile the wxWidgets source using VS2017 and a 64 bit build, this generates DLLs with the ending "vc_x64_custom.dll"
This means that "moneymanagerex/CMakeLists.txt" throws a bunch of errors in the following code..

        # check custom build name vs original one
        FOREACH(m_dll ${MMEX_WXDLLS})
            SET(m_dll "${wxWidgets_LIB_DIR}/${m_dll}")
            IF(EXISTS "${m_dll}_vc_mmex.dll")
                LIST(APPEND CMAKE_INSTALL_SYSTEM_RUNTIME_LIBS "${m_dll}_vc_mmex.dll")
            ELSEIF(EXISTS "${m_dll}_vc_custom.dll")
                LIST(APPEND CMAKE_INSTALL_SYSTEM_RUNTIME_LIBS "${m_dll}_vc_custom.dll")
            ELSEIF(EXISTS "${m_dll}_${WXDLLSUFF}")
                LIST(APPEND CMAKE_INSTALL_SYSTEM_RUNTIME_LIBS "${m_dll}_${WXDLLSUFF}")
            ELSE()
                MESSAGE(SEND_ERROR "${m_dll}_${WXDLLSUFF} not found")     <------
            ENDIF()
        ENDFOREACH()

Addition of 
		ELSEIF(EXISTS "${m_dll}_vc_x64_custom.dll")
                     LIST(APPEND CMAKE_INSTALL_SYSTEM_RUNTIME_LIBS "${m_dll}_vc_x64_custom.dll") 

Resolves the error.

Not sure why there is a check for "_vc_mmex_dll" as the code is processing WXDLLS ?

**Comment by slodki** [comment]
@grumpy235 Could you verify my PR? Could you build 32-bit version and check filename?

**Comment by grumpy235** [comment]
@slodki 
Will do.
Not exactly sure what this code does apart from copying the wxW DLLs into my build library "D:\Projects\mmex\build\install\bin" and then not using them as I still need to add wxW to my PATH variable otherwise I get "not found" messages.
I am guessing this is because it is building an install package which would normally go into a library which was always searched like the default "C:\Program Files\MoneyManagerEX"
You may need to modify the PR as I downloaded the pre-compiled wxW "wxMSW-3.1.1_vc141_x64_ReleaseDLL.7z" and the names there end "_vc141_x64.dll"
Now I am beginning to get my head around how CMake works (long way to go though !), I'll have a play with 32bit and 64bit both pre-compiled and locally compiled and get back to you in the next few days

**Comment by slodki** [comment]
The last checked name with `${WXDLLSUFF}` will manage your downloaded prebuild library.
`${WXDLLSUFF}` stores directory name with `_dll` replaced with `.dll`.
vc141_x64_dll\wxbase311u_xml_vc141_x64.dll = vc141_x64_dll\wxbase311u_xml_${WXDLLSUFF}

**Comment by grumpy235** [comment]
I have tried 

1. 32bit and 64bit wxW binaries with both the Dev and Release libraries smashed together
2. 32bit and 64bit source build.

For some reason CMake cannot find my wxW library so I had to specify "wxWidgets_ROOT_DIR" and "DwxWidgets_LIB_DIR" on my cmake command

Did not go as far as compiling and running mmex.

CMAKE_INSTALL_SYSTEM_RUNTIME_LIBS appears to be gets populated with the correct paths
